main.ps1

```diff
@@ -19,26 +21,67 @@ mkdir $dir | Out-Null
 Push-Location $dir
 
 # Download erlang
-"Downloading Erlang/OTP $version package"
-Invoke-WebRequest -Uri "https://packages.erlang-solutions.com/erlang/erlang/esl-erlang/FLAVOUR_1_general/esl-erlang_$version.0~windows_amd64.exe" -OutFile "esl-erlang_$version.0~windows_amd64.exe"
+"Downloading Erlang/OTP $otpVersion package"
+Invoke-WebRequest -Uri "https://packages.erlang-solutions.com/erlang/erlang/esl-erlang/FLAVOUR_1_general/esl-erlang_$otpVersion.0~windows_amd64.exe" -OutFile "esl-erlang_$otpVersion.0~windows_amd64.exe"
 
 # Install erlang (runs in background)
-"Installing Erlang/OTP $version"
-& ".\esl-erlang_$version.0~windows_amd64.exe" '/S'
+"Installing Erlang/OTP $otpVersion"
+& ".\esl-erlang_$otpVersion.0~windows_amd64.exe" '/S'
 
 # Wait for it..
 while (@(Get-Process | Where-Object { $_.name -match 'esl-erlang' }).length -gt 0) {
     Start-Sleep -Milliseconds 100
 }
 
+# Download rebar3
+if( $rebar3Version -eq 'true' )
+{
+    switch  -Wildcard ( $otpVersion )
+    {
+        '24*' { $rebar3Version='3.16.1' }
```

Comment:
Is there a place we can find the mapping between rebar versions and OTP versions? It would make find keeping this up to date a lot easier.

Reply:
@tsloughter or @ferd, sorry to drag you in, but we need your expertise. Do you know if there is a mapping somewhere between which Rebar3 version is compatible with which OTP version?

Reply:
There's only one I did one during the security announcement thing:

- 3.16.0+ (Erlang/OTP 22-24)
- 3.14.0 through 3.15.2 (Erlang/OTP 19-23)
- 3.11.0 through 3.13.3 (Erlang/OTP 18-22)
- 3.7.0 through 3.10.0 (Erlang/OTP 17-21)

There's nothing automated. Today we only support the current + 2 oldest OTP versions, but that won't help track which rebar3 versions match against which OTP versions, aside from combing over the changelogs.

Reply:
@ferd, thank you.
@lpil then I don't know how we can make maintenance of this easier. Luckily there is only one OTP-release per year, so if we skip all minors we can just add the latest rebar3-version every year. WDYT?